app/api/resources/mentorship_relation.py

```diff
@@ -232,6 +233,61 @@ def put(cls, request_id):
 
         return response
 
+@mentorship_relation_ns.route("mentorship_relation/<int:request_id>/task/<int:task_id>/comment/<int:comment_id>/report")
+class ReportViolationForComment(Resource):
+    @classmethod
+    @jwt_required
+    @mentorship_relation_ns.doc("report_violation_for_comment")
+    @mentorship_relation_ns.expect(auth_header_parser)
+    @mentorship_relation_ns.response(
+        HTTPStatus.OK, "%s" % messages.VIOLATION_WAS_REPORTED_SUCCESSFULLY
```

Comment:
Can you use f-strings in this file? As this is part of the coding standard we use for string interpolation. 
Here's the [coding standards page](https://github.com/anitab-org/mentorship-backend/wiki/Coding-Standards)
I know this file is not using this standard across, but that is in the plans for future issues.

Reply:
Done